docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -2,335 +2,447 @@
  "cells": [
```

Comment:
<p>It might help the reader to point out what you doing with the <code>args</code> method to scipy's <code>minimize</code> to fix the arguments that are not being optimized.  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Added